packages/entando-datatable/test/DataTable.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect';
-import { render } from '@testing-library/react';
+import { render, fireEvent, screen, createEvent, debug } from '@testing-library/react';
```

Comment:
please use `user-events` library instead of direct call to `fireEvents` when possible

Reply:
Hi @gidesan . There aren't any drag and drop events on `user-events` as I recalled on us discussing this in a related previous task. Though I can implement both if you want (use `fireEvent` on DnD-related functions, `user-events` on other events). Let me know what you think.

Reply:
Hi @gidesan I have made changes. pls review